src/resultspage.cpp

```diff
@@ -212,6 +230,24 @@ void ResultsPage::onJumpToCallerCallee(const Data::Symbol& symbol)
     ui->resultsTabWidget->setCurrentWidget(m_resultsCallerCalleePage);
 }
 
+void ResultsPage::setData(const Data::Symbol& data)
+{
+    m_resultsDisassemblyPage->setSymbol(data);
+}
+
+void ResultsPage::setData(const Data::DisassemblyResult& data)
```

Comment:
I also think that this shouldn't be a public function, it's only called internally, no? just inline the call below